components/dash-core-components/package.json

```diff
@@ -15,7 +15,7 @@
     "private::format.black": "black dash_core_components_base/ tests/ setup.py",
     "private::format.eslint": "eslint src --fix",
     "private::format.prettier": "prettier --config .prettierrc --write src/**/*.js",
-    "private::lint.black": "node -e \"if ((process.env.PYVERSION || 'python39') !== 'python36'){process.exit(1)} \" || black --check dash_core_components_base/ tests/ setup.py",
+    "private::lint.black": "node -e \"if ((process.env.PYVERSION || 'python310') !== 'python36'){process.exit(1)} \" || black --check dash_core_components_base/ tests/ setup.py",
```

Comment:
Oof, we're still running `black` in Py3.6? Really should move that up so we can use the latest `black`, which supports 3.8+. That can happen either here or when we drop 3.6 & 3.7, ie #2735.

Reply:
There's a bigger oof over in the 3.12 PR, you're not running pylint on your tests with any rules enabled

Reply:
How about using https://pre-commit.ci/ for black and flake8?

Reply:
We do have a pre-commit hook using husky (though I often end up skipping it as it’s kind of slow)